app/fileSender.js

```diff
@@ -18,13 +18,11 @@ async function sendPassword(file, authKey, rawAuth) {
     const xhr = new XMLHttpRequest();
     xhr.onreadystatechange = () => {
       if (xhr.readyState === XMLHttpRequest.DONE) {
+        const nonce = xhr.getResponseHeader('WWW-Authenticate').split(' ')[1];
+        file.nonce = nonce;
         if (xhr.status === 200) {
           return resolve(xhr.response);
         }
-        if (xhr.status === 401) {
```

Comment:
Do you want me to revert this removed code back. If in case the `status=401` then the below code would help to reset the `file.nonce`
```javascript
if (xhr.status === 401) {
 const nonce = xhr.getResponseHeader('WWW-Authenticate').split(' ')[1];
 file.nonce = nonce;
}
```